gcx/styles/src/partials/head-scripts.hbs

```diff
@@ -1,80 +1,25 @@
-                }
-                analytics.load = function (key, e) {
-                    var t = document.createElement("script");
-                    t.type = "text/javascript";
-                    t.async = !0;
-                    t.src =
-                    "https://cdn.segment.com/analytics.js/v1/" +
-                    key +
-                    "/analytics.min.js";
-                    var n = document.getElementsByTagName("script")[0];
-                    n.parentNode.insertBefore(t, n);
-                    analytics._loadOptions = e;
-                };
-                analytics._writeKey = "{{this}}";
-                analytics.SNIPPET_VERSION = "4.13.2";
-                analytics.load("{{this}}");
-                analytics.page();
-            }
-        })();
+    !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on","addSourceMiddleware","addIntegrationMiddleware","setAnonymousId","addDestinationMiddleware"];analytics.factory=function(e){return function(){if(window.analytics.initialized)return window.analytics[e].apply(window.analytics,arguments);var i=Array.prototype.slice.call(arguments);i.unshift(e);analytics.push(i);return analytics}};for(var i=0;i<analytics.methods.length;i++){var key=analytics.methods[i];analytics[key]=analytics.factory(key)}analytics.load=function(key,i){var t=document.createElement("script");t.type="text/javascript";t.async=!0;t.src="https://cdn.segment.com/analytics.js/v1/" + key + "/analytics.min.js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(t,n);analytics._loadOptions=i};analytics._writeKey="{{this}}";;analytics.SNIPPET_VERSION="4.16.1";
+    analytics.load("{{this}}");
+    analytics.page();
+    }}();
     </script>
     {{/with}}
     <script>window.uiRootPath = '{{{uiRootPath}}}'</script>
     <script  type="module" src="{{{uiRootPath}}}/js/vendor/two-columns/two-columns.esm.js"></script>
     <script  nomodule src="{{{uiRootPath}}}/js/vendor/two-columns/two-columns.js"></script>
-    
     <script  type="module" src="{{{uiRootPath}}}/js/vendor/gcx-contact-form/gcx-contact-form.esm.js"></script>
     <script  nomodule src="{{{uiRootPath}}}/js/vendor/gcx-contact-form/gcx-contact-form.js"></script>
- 
     <script  type="module" src="{{{uiRootPath}}}/js/vendor/gcx-schema/gcx-schema.esm.js"></script>
     <script  nomodule src="{{{uiRootPath}}}/js/vendor/gcx-schema/gcx-schema.js"></script>
-    <script  src="{{{uiRootPath}}}/js/vendor/enlighterjs/enlighterjs.min.js"></script>
\ No newline at end of file
+    <script  src="{{{uiRootPath}}}/js/vendor/enlighterjs/enlighterjs.min.js"></script>
```

Comment:
I added a blank line at the end of the file to make the HTML output more organized.